test/src/main/java/org/apache/accumulo/test/VolumeManagerIT.java

```diff
@@ -0,0 +1,148 @@
+import org.apache.accumulo.miniclusterImpl.MiniAccumuloConfigImpl;
+import org.apache.accumulo.test.functional.ConfigurableMacBase;
+import org.apache.accumulo.test.functional.ReadWriteIT;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.LocatedFileStatus;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.fs.RemoteIterator;
+import org.junit.jupiter.api.Test;
+
+public class VolumeManagerIT extends ConfigurableMacBase {
+
+  private String vol1;
+  private String vol2;
+
+  @Override
+  protected void configure(MiniAccumuloConfigImpl cfg, Configuration hadoopCoreSite) {
+    cfg.useMiniDFS(true);
+    cfg.setFinalSiteConfigUpdater((config) -> {
+      vol1 = config.getSiteConfig().get(Property.INSTANCE_VOLUMES.getKey());
+      assertTrue(vol1.contains("localhost"));
+      vol2 = vol1.replace("localhost", "127.0.0.1");
+      config.setProperty(Property.INSTANCE_VOLUMES.getKey(), String.join(",", vol2, vol1));
+
+      // Set Volume specific HDFS overrides
+      config.setProperty("general.volume.chooser",
+          "org.apache.accumulo.core.spi.fs.PreferredVolumeChooser");
+      config.setProperty("general.custom.volume.preferred.default", vol1);
+      config.setProperty("general.custom.volume.preferred.logger", vol2);
+
+      // Need to escape the colons in the custom property volume because it's part of the key. It's
+      // being
+      // written out to a file and read in using the Properties object.
+      String vol1FileOutput = vol1.replaceAll(":", "\\\\:");
+      String vol2FileOutput = vol2.replaceAll(":", "\\\\:");
+      config.setProperty(
+          Property.GENERAL_ARBITRARY_PROP_PREFIX.getKey() + vol1FileOutput + ".dfs.blocksize",
+          "10485760");
+      config.setProperty(
+          Property.GENERAL_ARBITRARY_PROP_PREFIX.getKey() + vol2FileOutput + ".dfs.blocksize",
+          "104857600");
+      config.setProperty(Property.GENERAL_ARBITRARY_PROP_PREFIX.getKey() + vol1FileOutput
+          + ".dfs.client.use.datanode.hostname", "true");
+      config.setProperty(Property.GENERAL_ARBITRARY_PROP_PREFIX.getKey() + vol2FileOutput
+          + ".dfs.client.use.datanode.hostname", "false");
+      config.setProperty(Property.GENERAL_ARBITRARY_PROP_PREFIX.getKey() + vol1FileOutput
+          + ".dfs.client.hedged.read.threadpool.size", "0");
+      config.setProperty(Property.GENERAL_ARBITRARY_PROP_PREFIX.getKey() + vol2FileOutput
+          + ".dfs.client.hedged.read.threadpool.size", "1");
+    });
+  }
+
+  @Test
+  public void testHdfsConfigOverrides() throws Exception {
+    try (AccumuloClient c = Accumulo.newClient().from(getClientProperties()).build()) {
+
+      Map<String,String> siteConfig = c.instanceOperations().getSiteConfiguration();
+      assertEquals("10485760", siteConfig
+          .get(Property.GENERAL_ARBITRARY_PROP_PREFIX.getKey() + vol1 + ".dfs.blocksize"));
+      assertEquals("104857600", siteConfig
```

Comment:
`104857600` is so similar to `10485760` that at first glance I thought both were the same.  Could replace `104857600` with something more visually distinct like `51200000`

Reply:
Updated test to use proposed value in e0c12c4.